### DISCUSSION TITLE
Feature request (new System Handle): OPC-UA Integration

### DISCUSSION THREAD

**Discussion by phuicy** [eProsima/Integration-Service#137] [Ideas]
Hi,

I just wanted to inquire to whether there was a plan to integrate OPC-UA?

thanks,
Guy

**Comment by rokel** [comment]
Simplest route might be to wrap the RTI OPC UA/DDS gateway once it's released.

**Comment by phuicy** [comment]
That will work to convert from OPC/UA -> DDS -> SOSS intermediate format. It would be nice to have a direct translation.
